JobConfig/ensemble/fcl/SamplingInput_sr0.fcl

```diff
@@ -0,0 +1,53 @@
+#include "Production/JobConfig/ensemble/fcl/prolog.fcl"
+
+#livetime: 60.000000
+#rue: 0.000000
+#dem_emin: 75.000000
+#tmin: 450.000000
+#run: 1201.000000
+#nevts: 2605
+
+
+process_name : SI
+
+source : { module_type : SamplingInput
+  inputCommands : @local::Ensemble.InputCommands
+  dropDescendantsOfDroppedBranches: false
+  run : 1201
+  subRun : 0
+  samplingSeed : 1
+  dataSets : { 
+      CeMLL: {
+        fileNames : ["/pnfs/mu2e/tape/phy-sim/dts/mu2e/CeMLeadingLog/MDC2024a_sm4/art/01/1d/dts.mu2e.CeMLeadingLog.MDC2024a_sm4.001210_00000691.art"]
+        weight : 1.088855e-06
```

Comment:
outside the scope of this PR we need to make these numbers come from the database, ala simefficiencies. They are campaign-dependent and we don't want to hardcode them in any script. Please create an issue for that after this PR is merged.

Reply:
this is just and example. one of my new scripts will make this fcl

Reply:
(I mean it will make the updated version)